docs/reference/troubleshooting/troubleshooting-unbalanced-cluster.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[troubleshooting-unbalanced-cluster]]
+== Troubleshooting an unbalanced cluster
+
+Elasticsearch is balancing shards across data tier to achieve a good compromise between:
+
+* shard count
+* disk usage
+* ingest load forecast for write indices
+
+Elasticsearch does not take into account amount or complexity of search queries when rebalancing shards.
+
+There is no guarantee that individual components are going to be evenly spread across the nodes.
+This might appear as some of the nodes have less shards allocated or using less disk
+as long as they are assigned with the shards with greater ingest load forecast.
```

Comment:
```suggestion
This could happen if some nodes have fewer shards, or are using less disk space, 
but are assigned shards with higher ingest load forecasts.
```

This sentence was quite hard to read, not sure if this edit makes it clearer. If not, consider breaking into smaller sentences.